### DISCUSSION TITLE
AML 1.5.1 doesn't launch

### DISCUSSION THREAD

**Discussion by ducki3x** [X2CommunityCore/xcom2-launcher#396] [Help]
AML 1.5.0 works fine, but AML 1.5.1 does nothing when launched (either normal or as admin); app never opens, a log file isn't even generated. Have tried placing the folder as a peer of AML 1.5.0, put it on desktop, tried fresh downloads, no joy. Any recommendations for next troubleshooting steps?

**Comment by RevZero** [comment]
Are you sure, that you extract all files from the archive?
Other that that, you could take a look at the windows event viewer, after starting AML. Its not unlikely, that windows will log some error there. Look in the category `Windows Logs\Application`.

![image](https://github.com/user-attachments/assets/fc5eb637-1fba-420c-b3b8-0cae7812ce3b)

**Comment by ducki3x** [comment]
Just deleted the folder and unzipped the package again, here's what I see - let me know if I'm missing something:

![AML1 5 1 Directory](https://github.com/user-attachments/assets/1aa58e8b-5314-4b60-99c9-0df99ff879e4)

Regarding the event viewer, I opened it up and tried launching 1.5.1 several times. Each time, the system did the same thing: my mouse cursor switched to a spinner for a couple of moments, then went back to being a pointer. Each time, I refreshed the application list in Event Viewer, but nothing new appeared:

![EventViewer](https://github.com/user-attachments/assets/bd2da3de-4c68-4746-a0f2-a5b486c562cc)

I also tried going back to an older package and forward to a new one. v1.4.1 loaded just fine for me, v1.6.0 alpha would not. For whatever reason, v1.5.0 is the last version that will run on my machine. I know of at least one other person having the same issue, the OP here:

https://www.reddit.com/r/xcom2mods/comments/1fg9fdt/trying_to_get_into_xcom_2_mods_but_aml_doesnt/

**Comment by RevZero** [comment]
Do you have some Antivir stuff running, which could be blocking AML?
You may also want to check if Windows is blocking something https://support.microsoft.com/en-us/windows/protection-history-f1e5fd95-09b4-46d1-b8c7-1059a1e09708

**Comment by ducki3x** [comment]
Fully disabled Norton, no change. Checked out the Windows Security protection history page, nothing blocked. I also tried the upgrade path on the app, extracting over a copy of my (working) 1.5.0 install, no luck there either.
